eZ Platform/Admin UI/USE_CASE.md

```diff
@@ -0,0 +1,25 @@
+# Work in Progress
+
+# Prototype: Hybrid Application
+## Goals:
+- Validate hypothesis of a hybrid application architecture.
+- Learn feasibility and difficulties
+
+## Stories
+
+### As a developer I want to simply reuse Admin Panel functionalities present in 1.x
+- ex: content type view
+
+### As a developer I want to simply reuse existing YUI-based modules present in 1.x
+- ex1: universal discovery widget
+- ex2: landing page editor
```

Comment:
this one is not mentioned in the previous document and since I have absolutely no clue on how it is done, it's a huge question mark for me.